util/journaldb/src/overlayrecentdb.rs

```diff
@@ -288,15 +288,19 @@ impl JournalDB for OverlayRecentDB {
         Box::new(self.clone())
     }
 
-    fn mem_used(&self) -> usize {
-        let mut mem = self.transaction_overlay.mem_used();
-        let overlay = self.journal_overlay.read();
-
-        mem += overlay.backing_overlay.mem_used();
-        mem += overlay.pending_overlay.heap_size_of_children();
-        mem += overlay.journal.heap_size_of_children();
+    fn get_sizes(&self, sizes: &mut BTreeMap<String, usize>) {
+        //sizes.insert(String::from("db_overlay_recent_transactions_size"), refs_size); //TODO MemoryDB
```

Comment:
Could you descibe with more detail what is this TODO, please?

Reply:
The current version of MemoryDB is just returning 0 on `mem_used()` and we don't have access to the underlying HashMap that is used in MemoryDB to at least get `len()`. The first intention for TODO is to change this when we update MemoryDB to new version, but now when I am thinking about it, we could just copy memory-db into project and add `lenI()` function.